osf_tests/test_guid.py

```diff
@@ -137,8 +137,9 @@ def setUp(self):
         self.node = NodeFactory()
 
     def test_resolve_guid(self):
-        res_guid = self.app.get(self.node.web_url_for('node_setting', _guid=True), auth=self.node.creator.auth)
-        res_full = self.app.get(self.node.web_url_for('node_setting'), auth=self.node.creator.auth)
+        with mock.patch('framework.csrf.handlers.get_current_user_id', return_value=self.node.creator_id):
+            res_guid = self.app.get(self.node.web_url_for('node_setting', _guid=True), auth=self.node.creator.auth)
+            res_full = self.app.get(self.node.web_url_for('node_setting'), auth=self.node.creator.auth)
```

Comment:
CSRF token will be different if we don't patch `framework.csrf.handlers.get_current_user_id` because these two requests use basic auth instead of cookie auth, resulting in different response body.